index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+   <link rel="stylesheet" type="text/css" href="style.css">
+   <meta charset="UTF-8">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <title>Portfolio Web Page </title>
+</head>
+
+<body>
+   <nav id="navbar">
+      <a href="#welcome-section">Contact</a>
+      <a href="#projects">Work</a>
+      <a href="#profile-link">About</a>
+   </nav>
```

Comment:
- [ ] Kindly  make sure that you phrased the **comments politely, as suggestions instead of orders** and **always includes an explanation, a "why", for each requested change.**For example:
- "Kindly I recommend you to include a `nav` tag inside the` header` tag to make your HTML code semantically correct and follows the best practice of HTML And Css."